### ISSUE TITLE
drupal documentation

### ISSUE THREAD

**Issue by kayiwa** [pulibrary/ops-catchall#69]
### Introduction

We talk about the webserver for drupal but it is missing the symbolic link creation required to run a successful `nginx -t` 

Also change the version of drupal to 10

### Acceptance criteria

- [ ] If there is no docs directory, create one.

- [ ] Update an existing markdown document or add a new one in the docs directory.

- [ ] It has introduction: Explains the goals and purpose of this documentation.

- [ ] It has a step by step guide: Provides the exact steps to guide the user on how to do a task.

#### Additional acceptance criteria 
- [ ] It includes references: References any related resources that have assisted creating this documentation (links to other tickets, online articles etc.).

- [ ] It includes workflow: Defines the steps in different environments (production, staging)

**Comment by acozine** [comment]
Do we still have any Drupal 10 applications any more? This might be obsolete.

**Comment by VickieKarasic** [comment]
Yes, this is obsolete - the last of the Drupal 10 apps (researchdata/PRDS) has migrated to campus hosting, so this can be closed.
